src/views/UsersList.vue

```diff
@@ -31,11 +33,19 @@
       :items="dataUsers"
       @changePage="onChangePage"
     ></jw-pagination>
+
+    <Modal
+      v-show="modal"
+      id="modal"
+      @closeModal="closeModal"
+      :dados="this.dadosPessoais"
```

Comment:
No template não usa o this, o this é usado só no script

Reply:
Alterei, não sabia que dava pra usar sem o this